src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -406,6 +406,17 @@ categories :
         use multi-nic hashing depending on what is detected on the system (e.g., number of NICs
         available, number of processes sharing the NICs).
 
+    - name        : MPIR_CVAR_CH4_OFI_HUGEPAGE_SIZE
+      category    : CH4
+      type        : int
+      default     : 2097152
+      class       : device
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_LOCAL
+      description : >-
+        Hugepage size is used to determine the am buffer size, so that the
+        buffer can be allocated at the page boundary.
```

Comment:
I think naming this hugepage size is a little misleading since it does not necessarily correspond to the operating system behavior. Still, using hugepage size as a guide for the AM buffer size is fine, so maybe a name like `MPIR_CVAR_CH4_OFI_MULTIRECV_BUFFER_SIZE` would be better and we can add a note to users that using hugepage size is good practice?